website/docs/reference/project-collaboration-mode.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Project collaboration mode
+---
+
+:::info Availability
+
+The change requests feature is an enterprise-only feature that was introduced in **Unleash 4.22.0**.
+
+:::
+
+Unleash project collaboration mode is a feature that allows to specify who can submit [change requests](change-requests.md). Currently, there are two modes:
+- **open** where everyone can submit change requests. This is a default setting for the existing and new Unleash projects.
+- **protected** where [admins, project members and owners](rbac.md#standard-roles) can submit change requests. Users who are not added to projects will not be able to submit change requests.
```

Comment:
Actually, maybe we could change the wording here. Project members and owners suggest that only these roles can submit change requests. That is not true in the implementation, during my testing I tried giving my viewer a custom role with the least privileges possible and I was allowed to submit CRs. There's a difference between being a project member (holding any role in the project) and having the project role of member / owner in the project.

Reply:
Do you suggest just saying project members?

Reply:
Yeah, I think that's an ok terminology. Maybe we also define it in this context, since we have the project member role. Something like `project members (any user holding a role in the project)`

Reply:
I thought that the following sentence "Users who are not added to projects will not be able to submit change requests." was clarifying this. WDYT?